Author: Holger Jaekel <holger.jaekel@gmx.de>
Summary: Use ANSI C functions like fopen() instead of 
         non-ANSI Posix functions like fopen64().
----

--- a/ofstd/include/dcmtk/ofstd/offile.h.orig
+++ b/ofstd/include/dcmtk/ofstd/offile.h
@@ -356,7 +356,7 @@
   {
     if (file_) fclose();
 #ifdef EXPLICIT_LFS_64
-    file_ = :: fopen64(filename, modes);
+    file_ = :: fopen(filename, modes);
 #else
     file_ = STDIO_NAMESPACE fopen(filename, modes);
 #endif
@@ -463,7 +463,7 @@
   {
 #if defined(EXPLICIT_LFS_64) && ! defined(__MINGW32__)
     // MinGW has EXPLICIT_LFS_64 but no freopen64()
-    file_ = :: freopen64(filename, modes, file_);
+    file_ = :: freopen(filename, modes, file_);
 #else
     file_ = STDIO_NAMESPACE freopen(filename, modes, file_);
 #endif
@@ -482,7 +482,7 @@
     if (file_) fclose();
 #if defined(EXPLICIT_LFS_64) && ! defined(__MINGW32__)
     // MinGW has EXPLICIT_LFS_64 but no tmpfile64()
-    file_ = :: tmpfile64();
+    file_ = :: tmpfile();
 #else
     file_ = STDIO_NAMESPACE tmpfile();
 #endif
@@ -806,7 +806,7 @@
 #else
 #ifdef HAVE_FSEEKO
 #ifdef EXPLICIT_LFS_64
-    result = :: fseeko64(file_, off, whence);
+    result = :: fseeko(file_, off, whence);
 #else
     result = :: fseeko(file_, off, whence);
 #endif
@@ -839,7 +839,7 @@
     offile_off_t result;
 #ifdef HAVE_FSEEKO
 #ifdef EXPLICIT_LFS_64
-    result = :: ftello64(file_);
+    result = :: ftello(file_);
 #else
     result = :: ftello(file_);
 #endif
@@ -863,7 +863,7 @@
     int result;
 #if defined(EXPLICIT_LFS_64) && ! defined(__MINGW32__) && ! defined(__QNX__)
     // MinGW and QNX have EXPLICIT_LFS_64 but no fgetpos64()
-    result = :: fgetpos64(file_, pos);
+    result = :: fgetpos(file_, pos);
 #else
     result = STDIO_NAMESPACE fgetpos(file_, pos);
 #endif
@@ -883,7 +883,7 @@
     int result;
 #if defined(EXPLICIT_LFS_64) && ! defined(__MINGW32__) && ! defined(__QNX__)
     // MinGW and QNX have EXPLICIT_LFS_64 but no fsetpos64()
-    result = :: fsetpos64(file_, pos);
+    result = :: fsetpos(file_, pos);
 #else
     result = STDIO_NAMESPACE fsetpos(file_, pos);
 #endif
